tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -0,0 +1,13 @@
+import torchvision
+from torchvision.models._utils import IntermediateLayerGetter
+
+
+class rigidRegistration:
```

Comment:
```suggestion
class RigidRegistration:
```

tiatoolbox/tools/registration/wsi_registration.py:5:8: N801 class name 'rigidRegistration' should use CapWords convention

Reply:
Resolve in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/372/commits/0a9fb74a704756ba8c2c5512de29ccd5b864674a